src/components/ListingTable/styles.module.scss

```
@@ -41,7 +41,9 @@
             padding: 16px 24px;
             overflow: hidden;
             border-radius: 0 0 var(--admin-table-radius) var(--admin-table-radius);
-            background: var(--admin-table-row-hover-surface, #f0edee);
+            // Same surface as the table body — no separate grey footer block.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)